ccmlib/scylla_cluster.py

```diff
@@ -96,13 +96,13 @@ def start(self, no_wait=False, verbose=False, wait_for_binary_proto=False,
                                profile_options=profile_options)
                 # Let's ensure the nodes start at different times to avoid
                 # race conditions while creating system tables
-                time.sleep(1)
+                time.sleep(0.1)
```

Comment:
this is without a loop its changing the behavior - not sure it will not cause tests to fail on slow machines

Reply:
The comment says it all - it is "to avoid race conditions when creating system tables". I can check with Tomek if there are any race conditions, but in any case, if there are, a sleep won't help. The test will be flaky anyway.